fdbserver/TLogServer.actor.cpp

```diff
@@ -1849,6 +1853,7 @@ Future<Void> tLogPeekMessages(PromiseType replyPromise,
 						if (sd.version >= reqBegin) {
 							firstVersion = std::min(firstVersion, sd.version);
 							const IDiskQueue::location end = sd.start.lo + sd.length;
+							ASSERT(end > sd.start); // DEBUG ASSERTION
```

Comment:
Prior to this assertion, why don't we do an ordinary check of this condition, and log the values of start, length, and end, so we have a little more info if this hits?